src/Avalonia.Controls/Primitives/Thumb.cs

```diff
@@ -47,6 +46,12 @@ static Thumb()
             remove { RemoveHandler(DragCompletedEvent, value); }
         }
 
+        internal void AdjustDrag(Vector v)
```

Comment:
This method may be useful outside of `Track` but I would like to give this technique a bit more testing before opening it up as a public API (probably would need a more descriptive name too, but I've been unable to think of one).